app/Facades/DivisionApi.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+namespace App\Facades;
+
+use Illuminate\Support\Facades\Facade;
+
+class DivisionApi extends Facade
+{
+    protected static function getFacadeAccessor()
+    {
+        return \App\Contracts\DivisionApiContract::class;
+    }
+}
```

Comment:
question: preference on the dependency injection vs facades? https://laravel.com/docs/10.x/facades#facades-vs-dependency-injection I think the explicit nature of defining the dependency where you need it instead of "getting" it everywhere is nicer, but it could go either way.

Reply:
Good point, I'll look into it. If we can `use DivisionAPI` where it's used, that's better. As we don't really need to activated everywhere.